server/src/items/classes/spell_scrolls/SpellScroll.js

```diff
@@ -0,0 +1,15 @@
+const Item = require("../Item");
+
+class SpellScroll extends Item {
+    onUsed() {
+        console.log("SpellScroll used")
+        super.onUsed();
+    }
+}
+
+
+SpellScroll.abstract = true;
+// Give all spell scrolls easy access to the list of magic effects.
+SpellScroll.prototype.MagicEffects = require("../../../gameplay/MagicEffects");
+
+module.exports = SpellScroll;
```

Comment:
This super class can is not needed for the other spell scrolls. I think they can just extend directly from the Item class.
The MagicEffects property can just be imported into that class as needed (only some of the spells use it), instead of being available to all spell scrolls..